src/test/amount_tests.cpp

```diff
@@ -37,6 +37,31 @@ BOOST_AUTO_TEST_CASE(GetFeeTest)
     BOOST_CHECK_EQUAL(feeRate.GetFee(999), 122);
     BOOST_CHECK_EQUAL(feeRate.GetFee(1e3), 123);
     BOOST_CHECK_EQUAL(feeRate.GetFee(9e3), 1107);
+
+    // With full constructor (allows to test internal precission)
+    feeRate = CFeeRate(KB, KB * 2);
```

Comment:
Please, improve and bike-shed the commit at will and I will probably squash. I warned about this, didn't I?